libs/community/langchain_community/callbacks/openai_info.py

```diff
@@ -24,6 +25,7 @@
     "gpt-4-32k-0613-completion": 0.12,
     "gpt-4-vision-preview-completion": 0.03,
     "gpt-4-1106-preview-completion": 0.03,
+    "gpt-4-0125-preview-completion": 0.03,
```

Comment:
Can you also add `gpt-4-turbo-preview-completion`, which always points to the latest gpt-4-turbo model